### DISCUSSION TITLE
zen4R work

### DISCUSSION THREAD

**Discussion by sje30** [codecheckers/codecheckers#58] [General]
I'm just getting back to grips with zen4R code.  Has anyone worked on this in the last six months since the new zen4R interface was released?  (Happy to do so myself.)

**Comment by sje30** [comment]
https://github.com/codecheckers/codecheck/blob/master/inst/extdata/templates/codecheck/codecheck-zenodo.R was last updated four years ago, so I assume not, and I'll take a stab at it.

**Comment by nuest** [comment]
No, done nothing. Great if you have a chance to update the workflow!
